{"name": "После оформления заказа его номер появляется в \"В работе\"", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <page_object.tests.order_feed_test.TestOrderFeedPage object at 0x1024a9430>, driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"5cbc3b5456ec5869449333fb458f7276\")>\nopen_stellar_burgers = None\n\n    @allure.title( 'После оформления заказа его номер появляется в \"В работе\"')\n    def test_order_in_progress(self, driver, open_stellar_burgers):\n        personal_account_page = PersonalAccountPage(driver)\n        main_page = MainPage(driver)\n        order_feed_page = OrderFeedPage(driver)\n    \n        personal_account_page.click_on_login_to_account()\n        personal_account_page.add_email()\n        personal_account_page.add_password()\n        personal_account_page.click_to_login_button()\n        main_page.scroll_to_ingredient()\n        main_page.add_ingredient_to_order()\n        main_page.add_bun_to_order()\n        main_page.checkout()\n        order_number = main_page.get_order_number()\n>       main_page.click_to_close_modal()\n\npage_object/tests/order_feed_test.py:110: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npage_object/pages/main_page.py:28: in click_to_close_modal\n    self.wait_new_url(Urls.MODAL_SPICY_X_SAUCE)\npage_object/pages/base_page.py:19: in wait_new_url\n    WebDriverWait(self.driver, 15).until(EC.url_to_be(url))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"5cbc3b5456ec5869449333fb458f7276\")>, method = <function url_to_be.<locals>._predicate at 0x10257fd80>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n../../PycharmProjects/Sprint_6/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException"}, "start": 1721553096680, "stop": 1721553119774, "uuid": "c3ec4bd0-1fc3-49ac-9d19-d24971acc1f2", "historyId": "891163114a89a20c88b2ae367fb223d4", "testCaseId": "891163114a89a20c88b2ae367fb223d4", "fullName": "page_object.tests.order_feed_test.TestOrderFeedPage#test_order_in_progress", "labels": [{"name": "parentSuite", "value": "page_object.tests"}, {"name": "suite", "value": "order_feed_test"}, {"name": "subSuite", "value": "TestOrderFeedPage"}, {"name": "host", "value": "MacBook-Pro-Ola.local"}, {"name": "thread", "value": "18239-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "page_object.tests.order_feed_test"}]}